### ISSUE TITLE
Logo should link you to the main page or perhaps the issues page

### ISSUE THREAD

**Issue by bcowgill** [kadirahq/react-storybook#110]
Clicking on the R E A C T  S T O R Y B O O K link at top left does nothing, should probably take you to the github issues page, storybook home page or be configurable.

**Comment by arunoda** [comment]
Yeah. Good one. Linking into this repo would be great.

**Comment by ritz078** [comment]
Sometimes when I publish storybook to showcase my component, I would like to rename `REACT STORYBOOK` to my component name and redirect it to its GitHub page. That should be optional.

**Comment by arunoda** [comment]
Currently, we have the original feature implemented on the latest version. 

@ritz078 I think we can file a new issue for this. We are now trying to change the architecture a bit. So, we need to bit before we can do this.
